typings.d.ts

```diff
@@ -1,10 +1,13 @@
 import { AsyncSeriesWaterfallHook } from "tapable";
+import { Compiler } from 'webpack';
 import { Options as HtmlMinifierOptions } from "html-minifier";
 
 export = HtmlWebpackPlugin;
 
 declare class HtmlWebpackPlugin {
   constructor(options?: HtmlWebpackPlugin.Options);
+
+  apply(compiler: Compiler): void;
```

Comment:
Possible alternative is to extend Plugin

```
import { Plugin } from 'webpack'; 
declare class HtmlWebpackPlugin extends Plugin {
  constructor(options?: HtmlWebpackPlugin.Options);
```

Haven't tested compilability, just a thought off the top of my head

Reply:
This caused some issues the last time we tried so I would rather go with the solution @MrStLouis proposed